src/collections/_documentation/platforms/python/default-integrations.md

```diff
@@ -23,6 +23,22 @@ This integration registers with the interpreter's except hook system.  Through t
 any exception that is unhandled will be reported to Sentry automatically.  Exceptions
 raised interactive interpreter sessions will not be reported.
 
+### Options
+
+You can pass the following keyword arguments to `ExcepthookIntegration()`:
+
+* `always_run`:
+    
+    ```python
+    #repl
```

Comment:
How about this?

```sh
$ python
>>> import sentry_sdk
>>> from sentry_sdk.integrations.excepthook import ExcepthookIntegration
>>> sentry_sdk.init(..., integrations=[ExcepthookIntegration(always_run=True)])
>>> raise Exception("I will become an error")
```

Reply:
I'm not good at the document. This is much better. :) Thank you for both correction. I'm updating now.